src/test/java/seedu/address/logic/parser/LinkCommandParserTest.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_ITEM;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_ITEM;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.LinkCommand;
+
+public class LinkCommandParserTest {
+
+    private LinkCommandParser parser = new LinkCommandParser();
+
+    @Test
+    public void parse_validArgs_returnsLinkCommand() {
+        assertParseSuccess(parser, " c/2 s/1", new LinkCommand(INDEX_SECOND_ITEM, INDEX_FIRST_ITEM));
```

Comment:
Perhaps can add in a test for multiple student indexes/class indexes?

Reply:
Good idea, added in!